enabling_redfish_subscriptions_on_bare_metal.adoc

```diff
@@ -0,0 +1,93 @@
+// This is included in the following assemblies:
+//
+// TBD
+
+:_content-type: PROCEDURE
+[id='enabling_redfish_subscriptions_on_bare_metal_{context}']
+= Enabling redfish subscriptions on bare metal
+include::TBD/enabling_redfish_subscriptions_on_bare_metal.adoc[]
+:context: TBD
+
+Users of bare metal hardware may want to receive events from the baseboard management controller (BMC) to act on them in case of a hardware fault, such as an increase in temperature or removal of a device. The Redfish standard includes the ability to subscribe to events, which will cause hardware events to be sent in a particular format to a destination URI. Configuring a subscription is done using the Metal3 API.
+
+Execute the following procedure from the desired node to configure event subscriptions and alerts.
+
+[IMPORTANT]
+====
+This feature only works on physical hardware running Redfish with support for events with a vendor interface set to `redfish` or `idrac-redfish`. 
+====
+
+.Procedure
+
+. Power off the bare metal node by using the baseboard management controller, and ensure it is off.
```

Comment:
I don't think it's required to power off the node using the baseboard management controller...
When using this feature the user will probably have their OCP Cluster running, the only thing they need to check would be if the node has support for Subscriptions by using the curl

Reply:
Addressed in modules/nw-rfhe-creating_bmc_event_sub.adoc.